decidim-core/decidim-core.gemspec

```diff
@@ -35,6 +35,7 @@ Gem::Specification.new do |s|
   s.add_dependency "invisible_captcha", "~> 0.10.0"
   s.add_dependency "jquery-rails", "~> 4.3"
   s.add_dependency "mini_magick", "~> 4.8"
+  s.add_dependency "loofah", "~> 2.0", ">= 2.2.1"
```

Comment:
We don't explicitly depend on this, and having it here might be confusing. Maybe you should add a comment specifying that there's a vulnerability on `2.2.0`?

Reply:
Done!

Reply:
Actually, I recently (yesterday) noticed that we're directly using a constant defined in the loofah `gem`, so we might want the dependency to stay after all.

Reply:
The comment about the vulnerability is good to have anyways! :+1: